### DISCUSSION TITLE
Copilot Error

### DISCUSSION THREAD

**Discussion by rglasgow-deloitte** [community/community#8886] [Copilot Conversations]
I am experiencing an error when using Copilot in conjunction with VSCode. This error only happens on the VSCode extension and not in the Webstorm (intelliJ) version. 
```
[INFO] [default] [2021-12-15T14:18:06.753Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[ERROR] [default] [2021-12-15T14:18:06.848Z] Request Error: self signed certificate in certificate chain
[ERROR] [ghostText] [2021-12-15T14:18:06.849Z] Error on ghost text request Error: self signed certificate in certificate chain
[INFO] [ghostText] [2021-12-15T14:18:06.849Z] Breaking, no choices
```

Are there any known work arounds?

As for my vscode setup. Fresh install of vscode, no .vscode settings on the machine. I have tried it in both insiders and stable versions of vscode. Additionally, I have also used the same github account on other computers with success.

**Comment by davecheney** [comment]
Please see https://github.com/github/feedback/discussions/8866

**Reply by timothyeckert** [reply]
There is no resolution in this result, just a suggestion that didn't work. Unsure how this is an approved "answer".
